website/docs/r/container_registry_task_schedule.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Container"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_registry_task_schedule"
+description: |-
+  Manages a Container Registry Task Schedule.
+---
+
+# azurerm_container_registry_task_schedule
+
+Manages a Container Registry Task Schedule.
```

Comment:
what is this resource doing? it takes a single argument and no properties?

Reply:
where is the schedule defined?

Reply:
@katbyte This resource is an operation-like resource, which "runs" when created. What it does is to schedule a task run for an existing `azurerm_container_registry_task`.

Reply:
should the resource be named `azurerm_container_registry_task_schedule_run_now` then as there is no "Schedule" and it just triggers a run now?

Reply:
@katbyte I've made the change. Meanwhile, I've removed the import test and document for this resource as it is an operation-like resource, it is meaningless to import it.

## Test

```shell
💢  TF_ACC=1 go test -v -timeout=20h ./internal/services/containers  -run='TestAccContainerRegistryTaskSchedule_'
=== RUN   TestAccContainerRegistryTaskSchedule_basic
=== PAUSE TestAccContainerRegistryTaskSchedule_basic
=== CONT  TestAccContainerRegistryTaskSchedule_basic
--- PASS: TestAccContainerRegistryTaskSchedule_basic (238.69s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/containers    238.705s
```